payment/service.go

```diff
@@ -102,6 +85,17 @@ func (s *Service) InitKafka() error {
 
 	_, logger := logging.SetupLogger(context.Background())
 
+	var err error
```

Comment:
should we deduplicate the two `InitKafka` functions?

Reply:
better in https://github.com/brave-intl/bat-go/pull/548/commits/af9b87ab8cf2e2ca208c3efdf1c658e4ec3f9b63